proposals/field-keyword.md

```diff
@@ -96,9 +96,13 @@ Set-only properties may also use `field`:
 
 ### Breaking changes
 
-The existence of the `field` contextual keyword within property accessor bodies is a potentially breaking change, proposed as part of a larger [Breaking Changes](https://github.com/dotnet/csharplang/issues/7964) feature.
+The existence of the `field` contextual keyword within property accessor bodies is a potentially breaking change.
 
-Since `field` is a keyword and not an identifier, it can only be "shadowed" by an identifier using the normal keyword-escaping route: `@field`. All identifiers named `field` declared within property accessor bodies can safeguard against breaks when upgrading from C# versions prior to 13 by adding the initial `@`.
+Since `field` is a keyword and not an identifier, it can only be "shadowed" by an identifier using the normal keyword-escaping route: `@field`. All identifiers named `field` declared within property accessor bodies can safeguard against breaks when upgrading from C# versions prior to 14 by adding the initial `@`.
+
+If a variable named `field` is declared in a property accessor, an error is reported.
+
+If an expression `field` would bind to a symbol with language version 13 or earlier, a warning may be reported.
```

Comment:
Why "may" here? Can we make it "will" and better describe the situation if necessary.

Would it be more clear to say something like

```suggestion
If a symbol named `field` is accessible in the body of a setter or getter that uses the `field` keyword, a warning will be reported. "Accessible" here means that a symbol is available that would be used for binding, if it did not conflict with the `field` keyword. 
```

Reply:
Thanks. Changed to "is" rather than "may be".

In terms of accessibility, we should state that it's accessible *at the reference to* `field`. For instance, the following is valid, even though the `field` local is accessible within (parts of) the accessor body.
```csharp
object Property
{
    get
    {
        if (field == null)
        {
            int @field = 0;
            return @field;
        }
        return field;
    }
}
```